sys/auto_init/saul/auto_init_si114x.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2017-2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     auto_init_saul
+ * @{
+ *
+ * @file
+ * @brief       Auto initialization of Si114x driver.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *              Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * @}
+ */
+
+#ifdef MODULE_SI114X
+
+#include "log.h"
+#include "saul_reg.h"
+#include "si114x.h"
+#include "si114x_params.h"
+
+/**
+ * @brief   Define the number of configured sensors
+ */
+#define SI114X_NUMOF    (sizeof(si114x_params) / sizeof(si114x_params[0]))
+
+/**
+ * @brief   Allocation of memory for device descriptors
+ */
+static si114x_t si114x_devs[SI114X_NUMOF];
+
+/**
+ * @brief   Memory for the SAUL registry entries
+ */
+static saul_reg_t saul_entries[SI114X_NUMOF * 4];
+
+/**
+ * @brief   Define the number of saul info
+ */
+#define SI114X_INFO_NUMOF    (sizeof(si114x_saul_reg_info) / sizeof(si114x_saul_reg_info[0]))
+
+/**
+ * @brief   Reference the driver structs
```

Comment:
Reminder: should be `@name`

Reply:
I'll fix this and the above comment tonight.

Reply:
Fixed.

Reply:
Just had a look at your changes and that's not what I was expecting. By the way, I updated the branch (force pushed)